README.md

```diff
@@ -170,64 +147,140 @@ The `bastion` does not attempt to manage user accounts. We suggest using [GitHub
+## Help
+
+**Got a question?**
+
+File a GitHub [issue](https://github.com/cloudposse/bastion/issues), send us an [email][email] or join our [Slack Community][slack].
+
+## Commerical Support
+
+Work directly with our team of DevOps experts via email, slack, and video conferencing. 
+
+We provide *commercial support* for all of our [Open Source][github] projects. As a *Dedicated Support* customer, you have access to our team of subject matter experts at a fraction of the cost of a fulltime engineer. 
+
+- **Questions.** We'll use a Shared Slack channel between your team and ours.
+- **Troubleshooting.** We'll help you triage why things aren't working.
+- **Code Reviews.** We'll review your Pull Requests and provide constructive feedback.
+- **Bug Fixes.** We'll rapidly work to fix any bugs in our projects.
+- **Build New Terraform Modules.** We'll develop original modules to provision infrastructure.
+- **Cloud Architecture.** We'll assist with your cloud strategy and design.
+- **Implementation.** We'll provide hands on support to implement our reference architectures. 
+
+## Community Forum
+
+Get access to our [Open Source Community Forum][slack] on Slack. It's **FREE** to join for everyone! Our "SweetOps" community is where you get to talk with others who share a similar vision for how to rollout and manage infrastructure. This is the best place to talk shop, ask questions, solicit feedback, and work together as a community to build *sweet* infrastructure.
+
 ## Contributing
 
-#### Bug Reports & Feature Requests
+### Bug Reports & Feature Requests
 
 Please use the [issue tracker](https://github.com/cloudposse/bastion/issues) to report any bugs or file feature requests.
 
-#### Developing
+### Developing
+
+If you are interested in being a contributor and want to get involved in developing this project or [help out](https://github.com/orgs/cloudposse/projects/3) with our other projects, we would love to hear from you! Shoot us an [email](mailto:hello@cloudposse.com).
 
-PRs are welcome. In general, we follow the "fork-and-pull" Git workflow.
+In general, PRs are welcome. We follow the typical "fork-and-pull" Git workflow.
 
  1. **Fork** the repo on GitHub
  2. **Clone** the project to your own machine
  3. **Commit** changes to your own branch
  4. **Push** your work back up to your fork
- 5. Submit a **Pull request** so that we can review your changes
+ 5. Submit a **Pull Request** so that we can review your changes
 
-NOTE: Be sure to merge the latest from "upstream" before making a pull request!
+**NOTE:** Be sure to merge the latest changes from "upstream" before making a pull request!
 
-## Change Log
+## Copyright
 
-View our closed [Pull Requests](https://github.com/cloudposse/bastion/pulls?q=is%3Apr+is%3Aclosed).
+Copyright © 2017-2018 [Cloud Posse, LLC](https://cloudposse.com)
 
 
-## Thanks
+## License 
 
-- [@neochrome](https://github.com/neochrome/docker-bastion), for providing a great basic bastion built on top of Alpine Linux
```

Comment:
We should add these to some section.

![image](https://user-images.githubusercontent.com/52489/42296544-e1232598-7fab-11e8-9cd3-e9e73fa51c1a.png)